planner/cascades/testdata/integration_suite_out.json

```diff
@@ -1079,6 +1079,19 @@
         "Result": [
           "1 1"
         ]
+      },
+      {
+        "SQL": "select /*+ MERGE_JOIN(t1) */ t1.b, t2.b from t1 inner join t2 on t1.a = t2.a;",
+        "Plan": [
+          "HashRightJoin_9 12500.00 root inner join, equal:[eq(test.t1.a, test.t2.a)]",
```

Comment:
It's still HashRightJoin 🤔

Reply:
cascades planner currently does not support hint. It's ok here.